apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -176,6 +176,8 @@ const ScheduleDirection = ({
     error: false
   });
 
+  const isSubwayRoute: boolean = route.type === 1 || route.type === 0;
```

Comment:
nitpick: Since this check is also used in other files, it might make sense to make this a function and move it to a helper

Reply:
That's a good point. https://github.com/mbta/dotcom/blob/master/apps/site/assets/ts/models/route.ts might be a good place for that.